swift/IceStorm/clock/README.md

```diff
@@ -1,168 +1,131 @@
-Overview
-========
+# IceStorm Clock
+
+## Overview
 
 This demo shows how to use [IceStorm][1].
 
 To run the demo, start the IceStorm service:
 
-```
+```shell
 icebox --Ice.Config=config.icebox
 ```
 
-This configuration assumes there is a subdirectory named db in the
-current working directory.
+This configuration assumes there is a subdirectory named db in the current working directory.
 
 In a separate window:
 
-```
-./Build/subscriber
+```shell
+swift run Subscriber
```

Comment:
Joe wants to switch to lowercase subscriber/publisher/client/server - for a follow-up PR.